package.json

```diff
@@ -33,15 +33,15 @@
     "@justeat/gulp-gh-pages": "^1.0.3",
     "@justeat/stylelint-config-fozzie": "^2.0.1",
     "assemble": "^0.24.3",
-    "autoprefixer": "^9.1.0",
-    "babelify": "^8.0.0",
+    "autoprefixer": "^9.1.5",
+    "babelify": "^10.0.0",
```

Comment:
v10 has been upgraded to use Babel 7, will this mean that modules will also have to update when they use this version of `gulp-build-fozzie`?

Reply:
Have bumped to v8 so that this indicates breaking change